core/src/main/java/dev/projectg/crossplatforms/config/ConfigManager.java

```diff
@@ -51,7 +56,8 @@ public ConfigManager(Path directory, Logger logger) {
             builder.registerExact(FormImage.class, new FormImageSerializer());
             builder.registerExact(CustomComponent.class, new ComponentSerializer());
```

Comment:
The BedrockForm and CustomComponent serializers are already registered in CrossplatForms.class if cumulus is available. The FormImage serializer also has to be moved there.